conan/tools/gnu/autotools.py

```diff
@@ -48,16 +48,18 @@ def make(self, target=None, args=None):
         str_args = self._make_args
         str_extra_args = " ".join(args) if args is not None else ""
         jobs = ""
-        if "-j" not in str_args and "nmake" not in make_program.lower():
+        if "-j" not in (str_args + str_extra_args) and "nmake" not in make_program.lower():
```

Comment:
This PR may be a good opportunity to provide a more robust test of `-j<number>` in arg. For example if args contains `DESTDIR=/user/smith-john/.conan/...`, conan doesn't inject -j